### DISCUSSION TITLE
Send full error on Telegram

### DISCUSSION THREAD

**Discussion by DyarWeb** [westacks/telebot#55] [Q&A] [answered]
Hello, I receive errors like
```
Laravel ERROR (local):
Unable to cast "NULL" variable to type "string". {"exception":"[object] (WeStacks\\TeleBot\\Exception\\TeleBotObjectException(code: 400): Unable to cast \"NULL\" variable to type \"string\". at /var/www/vendor /westacks/telebot/src/Exception/TeleBotObjectException.php:29)"} []
```
in Telegram, unfortunately, it is not clear where is the problem
How can send full error?

**Comment by punyflash** [comment, marked as answer]
You may customize [`telebot::log` ](https://github.com/westacks/telebot/blob/master/src/Laravel/views/log.blade.php) view to get full error trace. The view contains all default monolog's [`LogRecord`](https://github.com/Seldaek/monolog/blob/main/src/Monolog/LogRecord.php) properties as variables. To publish telebot's views, just execute command:

```bash
php artisan vendor:publish --provider="WeStacks\TeleBot\Laravel\Providers\TeleBotServiceProvider" --tag=views
```

But be aware that trace is quite long string and most likely will not fit in 1 telegram message and will be separated into 2-3 more messages.
